### PR TITLE
Update README for Picard metrics

### PR DESCRIPTION
**Pull request by wnojopra**

### CODE REVIEW

File: rna/README.md [hunk 1]
```diff
@@ -72,4 +72,22 @@ The `samples` table would then contain, for each sample:
 - [other samtools output files]
 - featureCounts_files (Array of STRINGS: paths to featureCounts output files)
 
-All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.
\ No newline at end of file
+All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.
```
Comment (mbookman) [inline rna/README.md]: I think this paragraph belongs before the step-by-step instructions:

Thus if you set each workflow to run from the sample entity and set the outputs to update the sample entity, you can run a progression of workflows. All of your workflow outputs will be stored in your Terra workspace bucket and the `sample` entity table will contain the paths to the outputs.

File: rna/README.md [hunk 2]
```diff
@@ -72,4 +72,22 @@ The `samples` table would then contain, for each sample:
...
+
+Additionally, you can run Picard tools to produce metrics on the BAM files.
```
Comment (mbookman) [inline rna/README.md]: All of the workflows are "optional", so I don't think we need to explicitly call this one out.

File: rna/README.md [hunk 3]
```diff
@@ -72,4 +72,22 @@ The `samples` table would then contain, for each sample:
...
+- salmon_quant_sf (STRING: path to Salmon output quant.sf file in GCS)
+- salmon_quant_genes_sf (STRING: path to Salmon output quant.genes.sf file in GCS)
+- [other Salmon output files]
+- STAR_bam (STRING: path to STAR [unsorted] output BAM in GCS)
+- [other STAR output files]
+- samtools_bam (STRING: path to samtools [sorted] output BAM in GCS)
+- [other samtools output files]
+- featureCounts_files (Array of STRINGS: paths to featureCounts output files)
+- rnaseq_metrics_path (STRING: path to RNA_Metrics file in GCS)
+- multiple_metrics_path (Array of STRINGS: paths to Multiple_metrics output files)
```
Comment (mbookman) [inline rna/README.md]: "in GCS"

**Review by mbookman — CHANGES_REQUESTED** [review]

**Review by mbookman — APPROVED** [review]
LGTM

**Review by ha-duong-technome — APPROVED** [review]

**Pull request merged**